cyclonedx_py/parser/environment.py

```diff
@@ -75,18 +75,23 @@ def __init__(self, use_purl_bom_ref: bool = False) -> None:
                 # Therefore, just go with a named license.
                 c.licenses.add(LicenseChoice(license_=License(license_name=i_metadata['License'])))
 
-            if 'Classifier' in i_metadata:
-                for classifier in i_metadata['Classifier']:
-                    # Trove classifiers - https://packaging.python.org/specifications/core-metadata/#metadata-classifier
-                    # Full list: https://pypi.python.org/pypi?%3Aaction=list_classifiers
-                    if str(classifier).startswith('License :: OSI Approved :: '):
-                        c.licenses.add(LicenseChoice(license_=License(
-                            license_name=str(classifier).replace('License :: OSI Approved :: ', '').strip()
-                        )))
-                    elif str(classifier).startswith('License :: '):
-                        c.licenses.add(LicenseChoice(license_=License(
-                            license_name=str(classifier).replace('License :: ', '').strip()
-                        )))
+            if sys.version_info >= (3, 8):
+               if sys.version_info >= (3, 10):
+                   classifiers = i_metadata.get('Classifier', [])
+               else:
+                   classifiers = [value for (key, value) in i_metadata._headers if key =='Classifier']    
```

Comment:
:x: 
if non of these is true, then `classifiers` woyuld be undefined.
but it is treated as an iterable

Reply:
Good catch, using python 3.7.15 I discovered that the code for python3.8 works in python3.7 too.

Reply:
Since python 3.7 is the last supported version, is it enough to add support for it or do we need another branch for other (untested) versions?